api/appointmentService.yaml

```diff
@@ -460,12 +502,18 @@ paths:
         - booking
       summary: Webhook subscriber
       operationId: processBookingWebhook
+      parameters:
+        - name: x-cal-signature-256
+          in: header
+          description: Webhook hmac
+          schema:
+            type: string
       requestBody:
         description: The json string provided by calcom
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/CalcomWebhookInput'
```

Comment:
since the refereces to CalcomWebhookInput is removed, do we need it anymore in the yaml?

Reply:
Yes, I would say so since the generated object is still used for deserializing the body content, after the HMAC check.